torchvision/models/detection/_utils.py

```diff
@@ -489,6 +489,8 @@ def _topk_min(input: Tensor, orig_kval: int, axis: int) -> Tensor:
     Returns:
         min_kval (Tensor): Appropriately selected k-value.
```

Comment:
nit: modify function description to reflect the signature
```suggestion
        min_kval (int): Appropriately selected k-value.
```